extension/packages/nextjs/app/eip-712/page.tsx

```diff
@@ -103,14 +108,27 @@ const Eip712: NextPage = () => {
   return (
     <div className="flex items-center flex-col flex-grow pt-10 px-8">
       <div className="flex flex-col gap-4 items-center">
+        <div className="text-center">
+          <h1 className="text-2xl font-bold">EIP-712</h1>
+          <div className="max-w-2xl">
+            EIP-712 defines a standard for hashing and signing typed structured
+            data in Ethereum. It enhances security and usability by enabling the
+            creation of more readable and secure signed messages, reducing the
+            risk of phishing attacks and user errors. For more details, visit
+            the{" "}
+            <Link
+              href="https://eips.ethereum.org/EIPS/eip-712"
+              className="underline font-bold text-nowrap"
+            >
+              EIP-712 specification
+            </Link>
+            .
```

Comment:
Let's add that they can find the code on `packages/nextjs/app/eip-712/page.tsx`. Similar to homepage, but maybe smaller.